extensions/interactions/MultipleChoiceInput/directives/multiple-choice-input-validation.service.ts

```diff
@@ -50,10 +50,10 @@ export class MultipleChoiceInputValidationService {
     var seenChoices = [];
     var numChoices = customizationArgs.choices.value.length;
 
-    if (numChoices < 4) {
+    if (numChoices < 2) {
       warningsList.push({
         type: AppConstants.WARNING_TYPES.CRITICAL,
-        message: 'Please enter at least four choices.'
+        message: 'Please enter at least two choices.'
       });
     }
```

Comment:
**Note:** From now forwards, we choose to only require at least four choices for Multiple Choice Input when the exploration is a curated lesson. For all other explorations, the minimum number of choices is two.